src/main/resources/rs117/hd/scene/areas.json

```diff
@@ -1927,18 +1927,16 @@
   {
     "name": "AL_KHARID_GRASS_FIXES",
     "aabbs": [
-      [ 3348, 3293, 0, 3415, 3329, 0 ],
-      [ 3399, 3267, 0, 3414, 3292, 0 ],
-      [ 3393, 3269, 0, 3398, 3292, 0 ],
-      [ 3389, 3274, 0, 3392, 3292, 0 ],
-      [ 3383, 3290, 0, 3385, 3292, 0 ],
-      [ 3358, 3292, 0, 3382, 3292, 0 ],
-      [ 3365, 3291, 0, 3375, 3291, 0 ],
-      [ 3316, 3328, 0, 3327, 3330, 0 ],
-      [ 3347, 3298, 0, 3347, 3314, 0 ],
-      [ 3266, 3262, 0, 3269, 3263, 0 ],
-      [ 3275, 3294, 0, 3275, 3295, 0 ],
-      [ 3292, 3328, 0, 3300, 3329, 0 ]
+      [ 3348, 3293, 3415, 3329 ],
```

Comment:
Some of these served no logical purpose. another i extended to correct an overworld bug.
Arguably could be renamed to "Mage Training Arena Grass" as short of the one fix that is all it is at this point.